src/bsoncxx/types.hpp

```diff
@@ -309,6 +314,12 @@ struct BSONCXX_API b_date {
         : value(std::chrono::duration_cast<std::chrono::milliseconds>(tp.time_since_epoch())) {
     }
 
+    ///
+    /// Default constructor for b_date.
+    ///
+    BSONCXX_INLINE b_date() : value(0) {
```

Comment:
Hmm. I just read his comment. I don't see why it would be uninitialized? std::chromo::milliseconds has a default constructor?